azure-mongodb.yml

```diff
@@ -194,6 +194,10 @@ provision:
     type: boolean
     default: true
     details: Whether or not public network access is allowed for this CosmosDB account.
+  - field_name: server_version
+    type: string
+    default: null
+    details: The default server version for mongodb
```

Comment:
I think this should just say: "The server version for MongoDB". Saying it's the default version implies that there's a way to override this.

Reply:
done, thanks!